parser.py

```diff
@@ -35,3 +35,18 @@ def parse_salary(raw_salary: str) -> Optional[float]:
         return float(replaced_salary)
     print('wrong salary data')
     return
+
```

Comment:
See line 18
Don't we get exception if there won't be 'span' in job_item? Maybe we can handle this chain(`.find().find().text`) more carefully?
Same for lines 19, 24

Reply:
done

Reply:
Seems that we still can have there the same exceptions.

Reply:
done